pandas/core/reshape/tile.py

```diff
@@ -26,69 +26,104 @@
 def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
         include_lowest=False):
     """
-    Return indices of half-open bins to which each value of `x` belongs.
+    Bin `x` and return data about the bin to which each `x` value belongs.
```

Comment:
I find "return data about the bin" a bit "wordy" without saying much. 
I suppose you want to be general because the return type can be either the bins, or the indices indexing into the bins? But I won't care about this: the summary line should be short and give an idea of the main use case. The extended summary can go more into detail (as you already do).

Starting from the above, I would cut that to the essential and just "Bin x", but that is maybe too short :-) 
What about "Convert continuous values into discrete bins" or "Discretize values in specified bins" or "Bin values in specified intervals", but not sure what is the best wording. 
@TomAugspurger any input?

Reply:
Yes, this should be simplified. "Bin values into discrete intervals"?

I don't 100% like "in specified intervals" as you don't have to specify the intervals.

Reply:
> "Bin values into discrete intervals"?

\+1, perfect combination of all my attempts :-)